package.json

```diff
@@ -682,6 +687,7 @@
     "cpy": "^8.1.1",
     "css-loader": "^3.4.2",
     "cssnano": "^4.1.11",
+    "csstype": "^3.0.2",
```

Comment:
Is this used?

Reply:
[a6cbf92](https://github.com/elastic/kibana/pull/159854/commits/a6cbf926ae28edc8671658061c55b60fbad519ed)